packages/pages/src/vite-plugin/build/closeBundle/closeBundle.ts

```diff
@@ -9,7 +9,7 @@ import {
   loadTemplateModules,
   TemplateModuleCollection,
 } from "../../../common/src/template/internal/loader.js";
-import { createFeaturesJson } from "../../../generate/features.js";
+import { createFeaturesJson } from "../../../generate/features/createfeaturesjson.js";
```

Comment:
could we make this file camelcase? git can struggle changing filename casing so you might need to do two commits or do like `git rm --cached` or something